### DISCUSSION TITLE
German sentences that should be supported

### DISCUSSION THREAD

**Discussion by balloob** [OHF-Voice/intents#348] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **German** sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by Skorfulose** [comment]
For covers, I usually say:

- Öffne/schließe den/die Rolladen in Pauls Zimmer
- Öffne/schließe den/die Rolladen im Wohnzimmer
- Öffne/schließe alle Rolladen
- Rollladen auf x Prozent
- Rollladen in Pauls Zimmer auf x Prozent
- Alle Rollladen öffnen/schließen
- Alle Rollladen hoch/runter

And others will interchangeably say „die Jalousie(n)“ instead of „Rolladen“

**Comment by envy** [comment]
I'm currently using the HomeKit integration witha Homepod and I frequently ask/tell Siri this:

- Wie ist die Außentemperatur? (What's the outside temperature? There is a sensor named "Außentemperatur")
- Schlafzimmer Süd auf 20%. (Bedroom south to 20%. There is a cover named "Schlafzimmer Süd" but I don't say Rolladen/Jalousie.)
- Schalte das Licht aus. (Turn off the lights. The Homepod knows it's in the bedroom and therefore only switched all lights in that room off)

**Comment by Scorpoon** [comment]
> For covers, I usually say:
> 
> * Öffne/schließe den/die Rolladen in Pauls Zimmer
> * Öffne/schließe den/die Rolladen im Wohnzimmer
> * Öffne/schließe alle Rolladen
> * Rollladen auf x Prozent
> * Rollladen in Pauls Zimmer auf x Prozent
> * Alle Rollladen öffnen/schließen
> * Alle Rollladen hoch/runter
> 
> And others will interchangeably say „die Jalousie(n)“ instead of „Rolladen“

Alles außer x Prozent ist dann mit #92 abgedeckt.

**Comment by kroimon** [comment]
> Alles außer x Prozent ist dann mit #92 abgedeckt.

"hoch/runter" für Cover fehlt

**Comment by ChristophHoltmann** [comment]
A few more granular ideas regarding covers:
"Öffne/Schließe die Rollläden\Jalousien ganz/komplett" for 0/100 % or "Öffne/Schließe die Rollläden/Jalousien halb/zur Hälfte" for 50 %
Even more precise would be something like: "Öffne/Schließe die Rollläden/Jalousien auf Schlitz/Spalt". The percentage might depend on the covers then..

**Comment by shurli** [comment]
Additionally to "Schalte ein/aus" i also would say "Dreh(e) auf/ab".

**Comment by jones1894** [comment]
Hi,
You should also take into account, that some would use "RollÄden" and some "RollAden" as the plural form of "Rolladen"

**Comment by jones1894** [comment]
And for getTemperature I'd like to use:
    Wie warm ist es (in/im) (der Küche/Schlafzimmer/draußen/oben etc.)?
and for setTemperature:
    Stell(e) die Heizung (in/im) (der Küche/Schlafzimmer/oben etc.) auf x Grad.
or even:
    Heize das Bad für's Duschen/Baden auf. (in this case you'd have to have a variable containing the precise temperature)

**Comment by geg2048** [comment]
Hi how proper should the grammar be and should we include slight dialect variations?

Because even tho according to the duden it is: 
-  das Rollo

it sounds weird for me so I use: 
-  **die** Rollo


Regarding dialects sometimes I say:
- moch

instead of:
- mach


Also regarding areas.
I often say:
- (bei (der | die | den)  | beim) {area}
for children rooms.

**Comment by ChristophCaina** [comment]
Another variation for getting a temperature value:

- "Wie viel Grad {Celsius | Fahrenheit} hat es <area>?"

Further more: Do we have other domains that needs to be added?
At the moment, I am running some Amazon Echos and have integrated different routines in the devices with which scripts will be triggered.
These will be used for changing settings on my EV Charging Station or start the charging of the EV...
But I could also imagine (for a wider adaptation) to controll TVs (turn on & set channel)...
Should we start with creating files for such examples, too? Or isn't that planned as of now?

**Reply by hillbicks** [reply]
I had the same question and asked on discord. I'll post the answer here. I just started digging into this and it seems to me that currently there is nothing to do, at least according to this page. 

https://home-assistant.github.io/intents/

the intent file for english doesn't contain more domains either, so I guess there is some other work to be done first. 

The status is of responses is also not clear to me since there is only one file for climate control which doesn't do much. Can I start working on those?

**Comment by ChristophCaina** [comment]
Covers können nicht nur Rollläden / Jalousien sein.
Es wären theoretisch auch Vorhänge oder aber Markiesen denkbar... diese könnten dann heraus oder herein gefahren werden.

Bei manchen Rollladen kann man dann auch noch einstellen, auf welchen Wert die Lammellen gekippt werden können...

**Comment by renarena** [comment]
```
        - "Wie hoch ist die Temperatur vom Thermostat <area>"
        - "Wie warm ist es <area>"

```
change this to something like

```
        - "Wie (hoch|tief) ist die Temperatur vom Thermostat <area>"
        - "Wie (warm|kalt) ist es <area>"
and doesnt we need something like` [in|im] <area>
```
